### ISSUE TITLE
BUG - Plugin doesn't start following v3.2.1 update

### ISSUE THREAD

**Issue by mthld** [standup-raven/standup-raven#250]
**Summary**
After uploading the last v3.2.1 release to Mattermost, the plugin doesn't start.

**Screenshots**
<img width="600" alt="Screenshot 2021-01-02 at 12 09 00" src="https://user-images.githubusercontent.com/41020854/103456096-4eef4680-4cf3-11eb-9318-c6b60993a3eb.png">

**Environment**
-  Mattermost version: 5.30.6
-  OS: Debian 10 Buster
-  Platform: web browser
-  Browser (if applicable): Safari
-  Platform Version: 14.0.2

Here is the server logs showing the error:
```
2021-01-02T12:06:32.556+0100 error mlog/log.go:229 Unable to activate plugin {"plugin_id": "standup-raven", "error": "GetChannel: Unable to find the existing channel., resource: Channel id: ms3tdp7dwi8jdnmcr7ph5khmph"}
2021-01-02T12:06:32.702+0100 error mlog/log.go:229 Unable to activate plugin {"plugin_id": "standup-raven", "error": "GetChannel: Unable to find the existing channel., resource: Channel id: ms3tdp7dwi8jdnmcr7ph5khmph"}
2021-01-02T12:06:32.876+0100 error mlog/log.go:229 Unable to activate plugin {"plugin_id": "standup-raven", "error": "GetChannel: Unable to find the existing channel., resource: Channel id: ms3tdp7dwi8jdnmcr7ph5khmph"}
2021-01-02T12:06:33.020+0100 error mlog/log.go:229 Unable to activate plugin {"plugin_id": "standup-raven", "error": "GetChannel: Unable to find the existing channel., resource: Channel id: ms3tdp7dwi8jdnmcr7ph5khmph"}
```

**Comment by mthld** [comment]
Still happening with v3.2.2:
```
2021-01-06T16:25:59.234+0100 error mlog/log.go:229 Unable to activate plugin {"plugin_id": "standup-raven", "error": "GetChannel: Unable to find the existing channel., resource: Channel id: ms3tdp7dwi8jdnmcr7ph5khmph"}
2021-01-06T16:25:59.372+0100 error mlog/log.go:229 Unable to activate plugin {"plugin_id": "standup-raven", "error": "GetChannel: Unable to find the existing channel., resource: Channel id: ms3tdp7dwi8jdnmcr7ph5khmph"}
2021-01-06T16:25:59.515+0100 error mlog/log.go:229 Unable to activate plugin {"plugin_id": "standup-raven", "error": "GetChannel: Unable to find the existing channel., resource: Channel id: ms3tdp7dwi8jdnmcr7ph5khmph"}
2021-01-06T16:25:59.663+0100 error mlog/log.go:229 Unable to activate plugin {"plugin_id": "standup-raven", "error": "GetChannel: Unable to find the existing channel., resource: Channel id: ms3tdp7dwi8jdnmcr7ph5khmph"}
```

**Comment by mthld** [comment]
Seems fixed by v3.2.2 recently released.
